cli/search/search.go

```diff
@@ -115,6 +126,52 @@ func GetVersionsFromGitRemote(repo string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// GetCommitsFromGitRemote returns commits list from specified remote git repo.
+func GetCommitsFromGitRemote(repo string) ([]string, error) {
+	commits := []string{}
+
+	if _, err := exec.LookPath("git"); err != nil {
+		return nil, fmt.Errorf("'git' is required for 'tt search' to work")
```

Comment:
This functions is public. How does it know that it is called from `tt search`?

Reply:
fixed

Reply:
It is not fixed. It still reports about `git` require for 'tt search' to work. How does this function know that it works with 'tt search' command?